### ISSUE TITLE
database.yml.example file not installed

### ISSUE THREAD

**Issue by anarcat** [sodabrew/puppet-dashboard#312]
The `FM` says:
1. Create an `/etc/puppet-dashboard/database.yml` file. Please see `/usr/share/doc/puppet-dashboard/examples/database.yml.example` for an example.

But then after installing the debian package:

```
/usr/share/doc/puppet-dashboard/examples/database.yml.example: No such file or directory
```

**Comment by bwitt** [comment]
hi @anarcat did you by chance install via deb package?

**Comment by bwitt** [comment]
if so https://github.com/sodabrew/puppet-dashboard/pull/353 should fix

**Comment by anarcat** [comment]
On 2018-08-21 23:53:39, Brian Witt wrote:
> hi @anarcat did you by chance install via deb package?

It's been so long I don't remember, but I believe so yes.

**Comment by bwitt** [comment]
Fixed via #353
